consensus/src/pending_votes.rs

```diff
@@ -211,6 +214,18 @@ impl PendingVotes {
 
         VoteReceptionResult::VoteAdded(voting_power)
     }
+
+    pub fn drain_votes(
+        &mut self,
```

Comment:
we can probably just use `self` instead of `&mut self`

Reply:
I don't want to copy, and incur any overhead,  so draining values before object as a whole is destroyed

Reply:
nit: it's better if you just consume `self` here, to avoid accidentally reusing `pending_votes` later after this call.

Reply:
I can do that only by using std::mem::swap, that seemed unsafe,  but it might actually be just a regular function. Will try separately